perfkitbenchmarker/pkb.py

```diff
@@ -143,8 +141,16 @@
     'scratch_disk_type', None,
     [disk.STANDARD, disk.REMOTE_SSD, disk.PIOPS, disk.LOCAL],
     'Type for all scratch disks. The default is standard')
+flags.DEFINE_string(
+    'data_disk_type', None,
+    'Type for all benchmark disks.')
```

Comment:
Define "data_disk" more crispy here, (or even consider provide a link here as code comment pointing to your design doc which should be uploaded as a wiki page on PKB). 

Also mention how we are going to handle the case where the provider does not support multiple disks beyond the root disk.